src/Types.tsx

```diff
@@ -0,0 +1,39 @@
+export type ResponseSectionsResults = {
```

Comment:
I would unify the way how you name the files. Some of them start with uppercase letter, some of them with lowercase letter. I would suggest to go for uppercase letter for files containing components and lowercase letters for other files.